KeyVault.Acmebot/Options/AcmebotOptions.cs

```diff
@@ -33,5 +33,7 @@ public class AcmebotOptions
         public GratisDnsOptions GratisDns { get; set; }
 
         public TransIpOptions TransIp { get; set; }
+
+        public DnsMadeEasyOptions DnsMadeEasy { get; set; }
```

Comment:
Please move the definition after `Cloudflare` (because it's in alphabetical order)